docs/userguide/autoscaling/cronfederatedhpa.md

```diff
@@ -0,0 +1,37 @@
+---
+title: CronFederatedHPA
+---
+
+In Karmada, the CronFederatedHPA is used for regular autoscaling actions. It can scale workloads that have a scale subresource or FederatedHPA.
+
+The typical scenario is to proactively scale up a workload in anticipation of predictable spikes in traffic. For example, if I know there will be a sudden increase in traffic every day at 9 AM, I want to scale up the related service ahead of time (e.g., 30 minutes prior) to handle the peak load and ensure uninterrupted availability.
+
+The CronFederatedHPA is implemented as both a Karmada API resource and a controller. The behavior of the controller is determined by the CronFederatedHPA resource. Running within the Karmada control plane, the CronFederatedHPA controller scales the workload's replicas or FederatedHPA's minReplicas/maxReplicas based on predefined cron schedules.
+
+## How does a CronFederatedHPA work?
+
+![cronfederatedhpa-architecture](../../resources/userguide/autoscaling/cronfederatedhpa-architecture.png)  
+Karmada implements CronFederatedHPA as a control loop that periodically checks the cron schedule time. If the scheduled time is reached, it scales the workload's replicas or FederatedHPA's minReplicas/maxReplicas.
+
+> Please note that this feature requires Karmada version v1.7.0 or later.
+
+## Scaling workloads with scale subresource
+
+CronFederatedHPA is capable of scaling workloads that have a scale subresource, such as Deployment and StatefulSet. However, it is important to note that there is a limitation. It is crucial to ensure that the scaling operation performed by CronFederatedHPA does not conflict with any other ongoing scaling operations. For example, if the workload is managed by both CronFederatedHPA and FederatedHPA simultaneously, the final result may be unexpected.  
+![autoscale-workload-conflicts](../../resources/userguide/autoscaling/autoscaling-conflicts.png)
+
+## Scaling FederatedHPA
+
+If you scale the workload directly using CronFederatedHPA, it may lose elasticity because even if the load is high, the replicas will remain the same. Therefore, a better choice would be to have CronFederatedHPA scale FederatedHPA first, and then have FederatedHPA scale workloads based on its metrics.
```

Comment:
> If you scale the workload directly using CronFederatedHPA, it may lose elasticity because even if the load is high, the replicas will remain the same.

People might don't understand why `the replicas will remain the same`, It would be better to add a brief description here.